rllib/core/models/torch/base.py

```diff
@@ -60,15 +60,11 @@ def _forward(self, inputs, **kwargs):
 
     """
 
-    def __init__(self, config: ModelConfig, skip_nn_module_init=False):
+    def __init__(self, config: ModelConfig):
```

Comment:
This seems superflous. Wasn't implemented anyways. I think we had already fixed the class hierarchy, such that this wou;dnt be necessary anymore.